s3tests_boto3/functional/__init__.py

```diff
@@ -150,6 +150,17 @@ def setup():
             )
     cfg.read(path)
 
+    #check for presence of IAM_section
```

Comment:
I think you misinterpreted what I tried to explain before. We want to check for these sections ONLY when those tests get run, which means the checks need to be in the test files themselves somehow not in __init__.py. 

The checks you had before were just fine, we don't need to add another section in the config file that says that another section in the config file exists.